pode.build.ps1

```diff
@@ -263,42 +314,55 @@ task Pack -If (Test-PodeBuildIsWindows) Build, {
     Copy-Item -Path ./src/Pode.Internal.psm1 -Destination $path -Force | Out-Null
     Copy-Item -Path ./src/Pode.Internal.psd1 -Destination $path -Force | Out-Null
     Copy-Item -Path ./LICENSE.txt -Destination $path -Force | Out-Null
-}, StampVersion, 7Zip, ChocoPack, DockerPack
+}, StampVersion, Compress, ChocoPack, DockerPack
 
 
 <#
 # Testing
 #>
 
 # Synopsis: Run the tests
-task Test Build, TestDeps, {
+Task TestNoBuild TestDeps, {
     $p = (Get-Command Invoke-Pester)
     if ($null -eq $p -or $p.Version -ine $Versions.Pester) {
         Remove-Module Pester -Force -ErrorAction Ignore
         Import-Module Pester -Force -RequiredVersion $Versions.Pester
     }
 
     $Script:TestResultFile = "$($pwd)/TestResults.xml"
-
+    # get default from static property
+    $configuration = [PesterConfiguration]::Default
+    $configuration.run.path = @('./tests/unit', './tests/integration')
+    $configuration.run.PassThru = $true
+    $configuration.TestResult.OutputFormat = 'NUnitXml'
+    $configuration.Output.Verbosity = $PesterVerbosity
+    $configuration.TestResult.OutputPath = $Script:TestResultFile
     # if run code coverage if enabled
     if (Test-PodeBuildCanCodeCoverage) {
         $srcFiles = (Get-ChildItem "$($pwd)/src/*.ps1" -Recurse -Force).FullName
-        $Script:TestStatus = Invoke-Pester './tests/unit', './tests/integration' -OutputFormat NUnitXml -OutputFile $TestResultFile -CodeCoverage $srcFiles -PassThru
+        $configuration.CodeCoverage.Enabled = $true
+        $configuration.CodeCoverage.Path = $srcFiles
+        $Script:TestStatus = Invoke-Pester   -Configuration $configuration
     }
     else {
-        $Script:TestStatus = Invoke-Pester './tests/unit', './tests/integration' -OutputFormat NUnitXml -OutputFile $TestResultFile -Show Failed -PassThru
+        $configuration.Output.Verbosity = 'Detailed'
```

Comment:
This overrides the verbosity value passed by the user, and set at lie 338 above.